### PR TITLE
add PHD role

### PR DESCRIPTION
**Pull request by giogio21**

### CODE REVIEW

**Review by lpatel99 — COMMENTED** [review]
Considering the issues we've had with espresso test recorder, I would recommend manually writing the tests here, especially considering they will be very simple.

**Review by lpatel99 — APPROVED** [review]
All good!

**Review by arnauds5 — COMMENTED** [review]
With this code, there is no difference between professor and PHD.
Is it planned to give different rights/etc to them or will they stay identical ?

**Comment by giogio21** [comment]
> With this code, there is no difference between professor and PHD.
> Is it planned to give different rights/etc to them or will they stay identical ?

Yes for the moment there is no difference but it plans to give PHD diffrents rights

File: app/src/androidTest/java/com/sdp13epfl2021/projmag/UserTypeChoiceTest.kt [hunk 1]
```diff
@@ -21,84 +21,29 @@ import org.junit.Rule
...
-                    0
-                ),
-                isDisplayed()
-            )
-        )
-        materialRadioButton.perform(click())
+    fun userProf() {
+        onView(withId(R.id.radioProfessorType)).perform(click())
         assertTrue(UserTypeChoice.isProfessor)
+    }
```
Comment (PaulCoral) [inline app/src/androidTest/java/com/sdp13epfl2021/projmag/UserTypeChoiceTest.kt:35]: these kind of tests using singletons are critcized in the code review should be changed, now or once we introduced dependency injections

**Review by PaulCoral — APPROVED** [review]
Seems ok, except my previous comment

**Review by arnauds5 — APPROVED** [review]

**Pull request merged**